astroquery/svo_fps/core.py

```diff
@@ -80,19 +78,17 @@ def data_from_svo(self, query, cache=True, timeout=None,
             # If no table element found in VOTable
             raise IndexError(error_msg)
 
-    def get_filter_index(self, wavelength_eff_min=0*u.angstrom,
-                         wavelength_eff_max=FLOAT_MAX*u.angstrom, **kwargs):
+    def get_filter_index(self, wavelength_eff_min, wavelength_eff_max, **kwargs):
```

Comment:
if you're interested in more cleanup here, there is no reason for using kwargs here, list them out instead, here and for the other methods, too.

Reply:
To be honest I am not interested in cleaning up `svo_fps` in the foreseeable future.